tasks/release.py

```diff
@@ -66,8 +66,20 @@ def print_shippable(ctx, quiet=False):
         diff_lines = get_diff(ctx, target, target_tag)
 
         # get the number of PRs that could be potentially released
+        # Only show the ones that have a changelog label that isn't no-changelog
         pr_numbers = parse_pr_numbers(diff_lines)
-        if pr_numbers:
+        all_no_changelog = True
+        for pr_num in pr_numbers:
+            try:
+                payload = get_pr(pr_num)
+                changelog_labels = get_changelog_types(payload)
+                if changelog_labels[0] != CHANGELOG_TYPE_NONE:
```

Comment:
Let's do `if changelog_labels and changelog_labels[0] != CHANGELOG_TYPE_NONE:` in case empty

Reply:
This was resolved in a PR after this :)